app/src/androidTest/java/com/github/h3lp3rs/h3lp/MainPageTestActivity.kt

```diff
@@ -0,0 +1,74 @@
+package com.github.h3lp3rs.h3lp
+
+import android.app.Activity
+import android.app.Instrumentation
+import android.content.Intent
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.matcher.IntentMatchers
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.Matchers
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class MainPageTestActivity {
+    @get:Rule
+    val testRule = ActivityScenarioRule(
+        MainPageActivity::class.java
+    )
+    @Test
+    fun clickingOnCPRButtonWorksAndSendsIntent() {
+        Intents.init()
+        val intent = Intent()
+        val intentResult = Instrumentation.ActivityResult(Activity.RESULT_OK, intent)
+        Intents.intending(IntentMatchers.anyIntent()).respondWith(intentResult)
+
+        onView(withId(R.id.CPR_rate_button)).perform(ViewActions.scrollTo(), click())
+
+        Intents.intended(
+            Matchers.allOf(
+                IntentMatchers.hasComponent(CprRateActivity::class.java.name))
+            )
+        Intents.release()
+    }
+
+    @Test
+    fun clickingOnProfileButtonWorksAndSendsIntent() {
+        Intents.init()
+        val intent = Intent()
+        val intentResult = Instrumentation.ActivityResult(Activity.RESULT_OK, intent)
+        Intents.intending(IntentMatchers.anyIntent()).respondWith(intentResult)
+
+        onView(withId(R.id.profile)).perform(click())
+
+        Intents.intended(
+            Matchers.allOf(
+                IntentMatchers.hasComponent(ProfileActivity::class.java.name))
+        )
+        Intents.release()
+    }
+
+    @Test
+    fun clickingOnHelpButtonWorksAndSendsIntent() {
+        Intents.init()
+        val intent = Intent()
+        val intentResult = Instrumentation.ActivityResult(Activity.RESULT_OK, intent)
+        Intents.intending(IntentMatchers.anyIntent()).respondWith(intentResult)
+
+        onView(withId(R.id.HELP)).perform(click())
+
+        Intents.intended(
+            Matchers.allOf(
+                IntentMatchers.hasComponent(HelpParametersActivity::class.java.name))
+        )
+        Intents.release()
+    }
```

Comment:
There seems to be some amount of code repetition here (which won't scale as we add the other activities and need to test them), you could refactor this testing code into a single method taking as attribute the button id and the class to which the intent should be directed. There would still be the problem that the CPR button will require scrollTo() which the others don't, to tackle this, I have two options:
1. Add a boolean in the general testing function, if it is set, scroll to the button, if not, only click it
2. Create 2 general testing functions, one for methods that will be on the scrollView, one for the ones that don't require scrolling to